exchange/exchange_test.go

```diff
@@ -146,12 +146,35 @@ func newRaceCheckingRequest(t *testing.T) *openrtb.BidRequest {
 }
 
 func TestPanicRecovery(t *testing.T) {
+	respStatus := 200
+	respBody := "{\"bid\":false}"
+	server := httptest.NewServer(mockHandler(respStatus, "getBody", respBody))
+	defer server.Close()
+
+	cfg := &config.Configuration{
+		CacheURL: config.Cache{
+			ExpectedTimeMillis: 20,
+		},
+		Adapters: blankAdapterConfig(openrtb_ext.BidderList()),
+	}
+
+	theMetrics := pbsmetrics.NewMetrics(metrics.NewRegistry(), openrtb_ext.BidderList())
+	e := NewExchange(server.Client(), nil, cfg, theMetrics, adapters.ParseBidderInfos("../static/bidder-info", openrtb_ext.BidderList()), gdpr.AlwaysAllow{}, currencies.NewRateConverterDefault()).(*exchange)
```

Comment:
I'd just use an empty http client here like `&http.Client{}` since this test is just testing the `recoverSafely()` method and that doesn't seem to be using it. This will help reduce the first 4 lines of the test and can make it a little simpler to read but will leave it up to you.

Reply:
You're right, this is a better way to write this test.  Change made.